### PR TITLE
feat: remove XMLA namespace prefix from XMLA handler [BACKLOG-40293]

### PR DESCRIPTION
**Pull request by loonatic**
@pentaho/millenniumfalcon please review

### CODE REVIEW

**Comment by buildguy** [comment]
### :x: Build failed in 4m 8s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox -DrunITs -pl mondrian
```
---
 :exclamation: No tests found!
 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
script returned exit code 1
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

File: mondrian/src/main/java/mondrian/tui/XmlaSupport.java [hunk 1]
```diff
@@ -555,17 +557,13 @@ public static Node[] extractNodesFromSoapXmla( byte[] bytes )
...
-    String xpath = getSoapXmlaRootXPath( xmlaPrefix );
+    String xpath = getSoapXmlaRootXPath( );
 
     // Note that this is SOAP 1.1 version uri
     String[][] nsArray = new String[][] {
       {
         SOAP_PREFIX, XmlaConstants.NS_SOAP_ENV_1_1
       },
-      {
-        xmlaPrefix, XmlaConstants.NS_XMLA
```
Comment (mbrasil) [inline mondrian/src/main/java/mondrian/tui/XmlaSupport.java]: We have to test if this is ok to remove.

File: mondrian/src/main/java/mondrian/tui/XmlaSupport.java [hunk 2]
```diff
@@ -582,13 +580,9 @@ public static Node[] extractNodesFromXmla( byte[] bytes )
 
   public static Node[] extractNodesFromXmla( Document doc )
     throws SAXException, IOException, XPathException {
-    final String xmlaPrefix = "xmla";
-    String xpath = getXmlaRootXPath( xmlaPrefix );
+    String xpath = getXmlaRootXPath( );
 
     String[][] nsArray = new String[][] {
-      {
-        xmlaPrefix, XmlaConstants.NS_XMLA
```
Comment (mbrasil) [inline mondrian/src/main/java/mondrian/tui/XmlaSupport.java]: same as above

**Comment by buildguy** [comment]
### :x: Build failed in 27m 29s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox -DrunITs -pl mondrian
```
---
 :exclamation: No tests found!
 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
<div align='center'>

[👍 Frogbot scanned this pull request and found that it did not add vulnerable dependencies.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 19m 47s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox -DrunITs -pl mondrian
```
---
:ok_hand: All tests passed!
 
Tests run: **3511**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/mondrian/job/PR-1369/3/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
### :x: Build failed in 35s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox -DrunITs
```
---
 :exclamation: No tests found!
 ---
### Errors:
<details><summary><b>Filtered log</b> (click to expand)</summary>
<p>

```
Command "git checkout -f 9aa234b6919b2c324e19ac2c531aae7bfeb3a289" returned status code 128:
stdout: 
stderr: fatal: reference is not a tree: 9aa234b6919b2c324e19ac2c531aae7bfeb3a289
```
 
</p>
</details>
 
 

:information_source:  _This is an automatic message_

**Comment by buildguy** [comment]
<div align='center'>

[👍 Frogbot scanned this pull request and found that it did not add vulnerable dependencies.](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

<details>
<summary> <b>Note:</b> </summary>


---
<div align='center'>

**Frogbot** also supports **Contextual Analysis, Secret Detection, IaC and SAST Vulnerabilities Scanning**. This features are included as part of the [JFrog Advanced Security](https://jfrog.com/advanced-security) package, which isn't enabled on your system.

</div>


</details>


---
<div align='center'>

[🐸 JFrog Frogbot](https://docs.jfrog-applications.jfrog.io/jfrog-applications/frogbot)

</div>

**Comment by buildguy** [comment]
### :white_check_mark: Build finished in 20m 21s
Build command:
```console
mvn clean verify -B -e -Daudit -Djs.no.sandbox -DrunITs -pl mondrian
```
---
:ok_hand: All tests passed!
 
Tests run: **3511**, Failures: **0**, Skipped: **0** &nbsp;&nbsp; [Test Results](https://wingman.orl.eng.hitachivantara.com/job/projects/job/mondrian/job/PR-1369/5/testReport)

 ---
 
 

:information_source:  _This is an automatic message_

**Review by afrjorge — APPROVED** [review]

File: mondrian/src/main/java/mondrian/tui/XmlaSupport.java (hunk 1 above)
Comment (mbrasil) [reply mondrian/src/main/java/mondrian/tui/XmlaSupport.java]: Tested and worked. We are good to go

**Review by mbrasil — APPROVED** [review]

**Pull request merged**